draft-ietf-quic-tls.md

```diff
@@ -552,7 +552,8 @@ older than 1.3 is negotiated.
 
 QUIC requires that the initial handshake packet from a client fit within the
 payload of a single packet.  The size limits on QUIC packets mean that a record
-containing a ClientHello needs to fit within 1171 octets.
+containing a ClientHello needs to fit within 1131 octets, though endpoints can
+reduce the size of their connection ID to increase by up to 22 octets.
```

Comment:
You'll of course need to change this if you ahnge the length encoding.

Reply:
Yeah, I'm close to killing this sentence, but it's somewhat useful to know where the limits are.